lib/util/platform.js

```diff
@@ -416,7 +416,10 @@ shaka.util.Platform = class {
       return navigator.userAgentData.platform.toLowerCase() == 'macos';
     }
     // Fall back to the old API, with less strict matching.
-    return navigator.platform.toLowerCase().includes('mac');
+    if (navigator.platform) {
+      return navigator.platform.toLowerCase().includes('mac');
+    }
+    return false;
```

Comment:
```suggestion
    if (!navigator.platform) {
      return false;
    }
    return navigator.platform.toLowerCase().includes('mac');
```

Reply:
@MichaelSweden does it work in Cobalt?

Reply:
Yes it works in Cobalt. I have tested an older version on ARM platform and latest main branch in PC Xubuntu.